components/Navigation/navigation.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Navigation</title>
+    <link rel="stylesheet" href="../Navigation/navigation.css" />
+    <link
+      rel="stylesheet"
+      href="/components/Navigation/navigation-media-query.css"
+    />
+    <script
+      src="https://kit.fontawesome.com/e5dbe6b99b.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+  <main>
+    <header>
+      <section class="flex nav-section ">
+        <div class="logo-container">
+          <a id="btnHamburger" href="#" class="header-toggle menu-btn icon">
+            <span></span>
+            <span></span>
+            <span></span>
```

Comment:
There are multiple empty spans is it supposed to be like this??

Reply:
Yes, its for hamburger menu. I actually created it, not used icons

Reply:
fine then you can merge.